### PR TITLE
Add @BulandMalik bio

### LINKED ISSUE #13: Add bio for BulandMalik
- [x] create a new branch
- [x] add bio file for Buland Malik
- [x] create pull request
- [ ] Get feedback
- [ ] merge pull request

### PR DESCRIPTION
**Pull request by BulandMalik**
Closes #13

### CODE REVIEW

File: bulandmalik.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Student
+
+* Name: Buland Malik
+* Location: Santa Clara, CA
```
Comment (BulandMalik) [inline bulandmalik.md:4]: How long you have been living there?

Comment (BulandMalik) [inline bulandmalik.md:4]: 8 years now :+1:

File: bulandmalik.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+* Username: @BulandMalik
```
Comment (gauravbansal17) [inline bulandmalik.md:6]: What did you had for breakfast?

File: bulandmalik.md (hunk 1 above)
Comment (BulandMalik) [inline bulandmalik.md:4]: :+1:

File: bulandmalik.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# GitHub Student
+
+* Name: Buland Malik
+* Location: Santa Clara, CA
+* Time: 8 years
```
Comment (BulandMalik) [inline bulandmalik.md:5]: :+1:

**Comment by BulandMalik** [comment]
looks good, let's merge! :+1: smile :exclamation:

**Pull request merged**